content/en/runtime.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Runtime
+status: Feedback Appreciated
+category: concept
+tags: ["application", "", ""]
+---
+
+## What it is
+
+A runtime, in general, is something that executes a piece of software.
+It is an abstraction of the underlying operating system that knows how to translate the program's commands into respective actions on the operating system level. 
+
+In the context of [cloud native](/cloud-native-apps/), we refer to the container runtime when we talk about the _runtime_. 
+A container runtime specifically implements the [Open Container Initiative](https://opencontainers.org/) specification to ensure consistent handling around the ecosystem. 
```

Comment:
What does "consistent handling around the ecosystem" mean?

Reply:
it means that the containers follow the OCI Specified rules that allows containers to work smoothly with different types of runtimes

Reply:
@CathPag is it more clear now, also taking the following text into consideration?

Reply:
Yeah, I don't like tech stack, but ecosystem sounds wrong. Ecosystem is more than just technology and I think you refer to the technologies on the landscape. Maybe a different word?

Reply:
After thinking about it, maybe Container Orchestrators might be suitable.
```suggestion
A container runtime specifically implements the [Open Container Initiative](https://opencontainers.org/) specification to ensure consistent handling around different container orchestration technologies. 
```